bundles/binding/org.openhab.binding.diyonxbee/.classpath

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" output="target/classes" path="src/main/java"/>
+	<classpathentry excluding="**" kind="src" output="target/classes" path="src/main/resources"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="lib" path="lib/log4j.jar"/>
+	<classpathentry kind="lib" path="lib/RXTXcomm.jar"/>
+	<classpathentry kind="lib" path="lib/xbee-api-0.9.jar" sourcepath="/home/juergen/code/home.automation/xbee"/>
```

Comment:
Hi,
Bro thank you for the reply and code.This is diyonxbee binding right.I need
the code which is there in this link
http://code.google.com/r/diaoulael-xbee/source/browse
<http://code.google.com/r/diaoulael-xbee/source/browse>.wether both are
same?.
I am very very new to opehab and currently I am trying to do the project
using the xbee binding. While searching i got this link "
http://technomindshare.blogspot.in/2014/04/interfacing-xbee-with-openhab.html"
   so i am trying to do this project.But I am
stuck there I am unable to get xbee binding shown in that link.

I think you have done the openhab project using xbee binding using
eclipse ide, so can you please give me a documentation how you have
done


On Thu, Feb 4, 2016 at 7:34 PM, robnielsen <notifications@github.com> wrote:

> In bundles/binding/org.openhab.binding.diyonxbee/.classpath
> <https://github.com/openhab/openhab/pull/3965#discussion_r51875578>:
>
> > @@ -0,0 +1,11 @@
> > +<?xml version="1.0" encoding="UTF-8"?>
> > +<classpath>
> > +	<classpathentry kind="src" output="target/classes" path="src/main/java"/>
> > +	<classpathentry excluding="**" kind="src" output="target/classes" path="src/main/resources"/>
> > +	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
> > +	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
> > +	<classpathentry kind="lib" path="lib/log4j.jar"/>
> > +	<classpathentry kind="lib" path="lib/RXTXcomm.jar"/>
> > +	<classpathentry kind="lib" path="lib/xbee-api-0.9.jar" sourcepath="/home/juergen/code/home.automation/xbee"/>
>
> You'll need to fix sourcepath
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openhab/openhab/pull/3965/files#r51875578>.
>